test/feature/win/agent_config.json

```diff
@@ -0,0 +1,84 @@
+{
```

Comment:
Generate a json configuration that is used for [CloudWatchAgent EC2 Mac](https://github.com/aws/amazon-cloudwatch-agent/tree/main/translator/translate/metrics)